internal/pkg/service/buffer/storage/config.go

```diff
@@ -0,0 +1,40 @@
+package storage
+
+import (
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/level/local"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/level/staging"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/level/target"
+)
+
+// Config contains default configuration for the storage.
+// Default settings can be customized in definition.TableSink.
+type Config struct {
+	Local   local.Config   `configKey:"local"`
+	Staging staging.Config `configKey:"staging"`
+	Target  target.Config  `configKey:"target"`
+}
```

Comment:
`storage.Config` sa sklada z konfiguracii jednodlivých `storage.Level`.

Teda v každen "doméne" je aj konfiguračná štruktúra, ktorá sa danej domény týka, a potom sa to iba poskladá do celkovej konfigurácie.